c/tskit/tables.h

```diff
@@ -697,28 +697,22 @@ typedef struct {
 /* Leave room for more positive check flags */
 #define TSK_NO_CHECK_POPULATION_REFS (1 << 12)
 
-/* Flags for load tables */
-#define TSK_BUILD_INDEXES (1 << 0)
+/* These flags are for table collection load or init, or used as
+   flags on table collection or individual tables.
+ * As flags are passed though from load to init they share a namespace */
+#define TSK_LOAD_SKIP_TABLES (1 << 0)
+#define TSK_TABLE_NO_METADATA (1 << 1)
+#define TSK_TC_SKIP_REFERENCE_SEQUENCE (1 << 2)
+#define TSK_TC_NO_EDGE_METADATA (1 << 3)
+#define TSK_TS_INIT_BUILD_INDEXES (1 << 4)
```

Comment:
Doesn't TSK_TS_INIT_BUILD_INDEXES belong in trees.h though, and live in a separate namespace? I don't think it interacts with the TableCollection of Table inits at all.

Ah, but SKIP_TABLES and SKIP_REFSEQ can also be passed to ``tsk_treeseq_load``, right? So namespacing these as LOAD makes sense, maybe?

Reply:
Yes, this is the issue - all flags that get passed to `tsk_treeseq_load` get passed to `tsk_table_collection_load` and then all passed on again to `tsk_table_collection_init` or `tsk_table_collection_loadf_inited`. So they all need to be in this block.

You're right though `TSK_TS_INIT_BUILD_INDEXES` is unrelated - got confused with tsk_tc_init. Will move that.

Reply:
Will change the two "skips" to "loads" also as that makes more sense to me.